### PR TITLE
INDY-941: fix agents

### PR DESCRIPTION
**Pull request by ashcherbakov**

### CODE REVIEW

File: indy_client/agent/run_agent.py [hunk 1]
```diff
@@ -64,7 +64,7 @@ def runAgent(agent, looper=None, bootstrap=None):
 
     def is_connected(agent):
         client = agent.client
-        if not client.can_send_write_requests():
+        if (client.mode is None) or (not client.can_send_write_requests()):
```
Comment (andkononykhin) [inline indy_client/agent/run_agent.py]: I think it makes sense to check not defined yet mode inside can_send_... methods.

Comment (ashcherbakov) [reply indy_client/agent/run_agent.py]: I agree, I just wanted to make a hot fix here. I will send a separate PR to move this condition to can_send_write_requests()

Comment (andkononykhin) [reply indy_client/agent/run_agent.py]: i see

**Comment by ashcherbakov** [comment]
test this please

**Comment by ashcherbakov** [comment]
test this please

**Pull request merged**